flow/project.py

```diff
@@ -84,12 +84,12 @@
 {template_dir}
 
 All template variables can be placed within a template using the standard jinja2
-syntax, e.g., the project root directory can be written like this: {{ project._rd }}.
+syntax, e.g., the project root directory can be written like this: {{{{ project._rd }}}}.
```

Comment:
This may appear to be subtle -- we want to render two literal braces, like `{{ project._rd }}`. However, in strings designed for use with `str.format()`, it is necessary to include two literal braces for one brace to appear in the result.

Example:
```python
>>> "{{ project._rd }}, stuff, hello {world}".format(world='earth')
'{ project._rd }, stuff, hello earth'
>>> "{{{{ project._rd }}}}, stuff, hello {world}".format(world='earth')
'{{ project._rd }}, stuff, hello earth'
```